packages/fxa-payments-server/package.json

```diff
@@ -54,7 +54,7 @@
     "@babel/core": "^7.15.0",
     "@fluent/bundle": "^0.18.0",
     "@fluent/langneg": "^0.6.2",
-    "@fluent/react": "^0.13.1",
+    "@fluent/react": "^0.14.1",
```

Comment:
fluent/react incorrectly declared the package to be commonjs even though the code was in esm.  (But I didn't upgrade to latest due to https://mozilla-hub.atlassian.net/browse/FXA-7458.)

Reply:
I actually did end up updating Storybook stories and frontend tests for the Fluent upgrade.  So I suspect that we can upgrade to the latest Fluent with those changes, but I'm not going to do that in this PR.